### DISCUSSION TITLE
On copy file

### DISCUSSION THREAD

**Discussion by eosynergy-gb** [mihaimoga/IntelliFile#2] [Ideas]
On copy (F5) file,  copy immediately without asking, you should first ask before you make the acicon and the active disk unit indicator should be displayed at the top, not in the bottom bar.

**Comment by cleasil** [comment]
Buonasera, sono dello stesso avviso di "osynergy-it" una immediata visibilità se il percorso viene indicato in alto in ogni finestra.
Forse il programma sarà più utilizzato se vi è anche un modulo di traduzione dei comandi e delle istruzioni.

**Comment by ecs-molty** [comment]
## ✅ Workflow Infrastructure Complete

All three pieces from this discussion are now in place:

1. **AGENTS.md** — workflow section added in `41230f8b`. Mandatory 5-step process (Observe → Diagnose → Implement → Verify → Match), evidence requirements table, testing-without-deploying patterns.

2. **GitHub Labels** — all 5 `workflow:` labels already existed:
   - `workflow: problem-observed`
   - `workflow: diagnosis-complete`
   - `workflow: implemented`
   - `workflow: verified`
   - `workflow: needs-human`

3. **Issue Template** — `Desired Outcome` + `Verification Method` fields added in `6c568298` (March 23).

All sub-agents working in this repo will now receive the workflow instructions automatically via AGENTS.md. The Hudu replacement proving ground (discussion #5153) can proceed with this foundation in place.
